nodes/clarify_util.js

```diff
@@ -92,15 +94,19 @@ module.exports = {
 
     let validationErrors = [];
     validateString(validationErrors, 'name', name);
-    validateString(validationErrors, 'type', type);
+    validateStringEnum(validationErrors, 'type', type, ['enum', 'numeric']);
     validateString(validationErrors, 'description', description);
     validateMapStringWithStrings(validationErrors, 'labels', labels);
-    validateMapStringWithStrings(validationErrors, 'annotations', annotations);
+    validateMapStringWithString(validationErrors, 'annotations', annotations, [
+      'aggregation',
+      'measurement',
+      'prediction',
+    ]);
```

Comment:
Part of the previous misunderstanding perhaps?

These arguments are _not_ for the annotations field, but for the `sourceType` field (as you have already added below).

Reply:
No misunderstanding here, just a plain old bug. I have updated the last commit.